HW03/zayats_aliaksei_hw03_t_04.rb

```diff
@@ -0,0 +1,15 @@
+def symbol_count(string)
+  letters = 0
+  digits = 0
+  digit_pattern = /[0-9]/
+  letter_pattern = /[a-zA-Z]/
+  string.split("").each do |symbol|
+    if symbol.match(digit_pattern)
+      digits += 1
+    elsif symbol.to_s.match(letter_pattern)
+      puts symbol
+      letters += 1
+    end
```

Comment:
We can try to use a shorter option 🙂 
```suggestion
    digits += 1 if symbol.match(digit_pattern)
    letters += 1 if symbol.match(letter_pattern)
```